### ISSUE TITLE
Invalid props.style key supplied to `View`

### ISSUE THREAD

**Issue by Lughino** [shoutem/ui#88]
I get this warning:
```javascript
Warning: Failed prop type: Invalid props.style key `shoutem` supplied to `View`.
Bad object: {
  "shoutem": {
  ....
```
```javascript
Warning: Failed prop type: Invalid props.style key `shoutem` supplied to `RCTView`.
Bad object: {
  "shoutem": {
  ....
```
Tried with projects from scratch, both react-native 0.36 and 0.37.

The steps to reproduce:
* `react-native init testwarningshoutem`
* `yarn add @shoutem/ui`

```javascript
// styles.js
module.exports = {
  test: {
    backgroundColor: '#ccc',
    border: 2,
  }
};
```

```javascript
// dummy index.ios.js
import React, { Component } from 'react';
import {
  AppRegistry,
  StyleSheet,
} from 'react-native';
import {
  Row,
  Subtitle,
  Caption,
  Text,
  TouchableOpacity,
  View,
} from '@shoutem/ui';
import styles from './styles';

export default class testwarningshoutem extends Component {
  render() {
    return (
      <Row style={styles.test}>
          <View styleName="vertical">
            <View styleName="horizontal space-between">
              <Subtitle
                styleName="">
                sdfdsfdsfdsfds
              </Subtitle>
              <Caption>
                10/10/2016
              </Caption>
            </View>
            <Text>
              dffdgdfgxxxfrfrgegre
            </Text>
          </View>
        </Row>
    );
  }
}

AppRegistry.registerComponent('testwarningshoutem', () => testwarningshoutem);
```
The styles is applied properly, but it's not deleted the property `shoutem` and generates this warning.

**Comment by pvpshoot** [comment]
+1

**Comment by pvpshoot** [comment]
any progress?

**Comment by pvpshoot** [comment]
@Lughino try to upgrade to 0.37

**Comment by buhe** [comment]
+1

**Comment by SoHotSoup** [comment]
Could you please verify this is fixed with latest `@shoutem/ui`, version `v0.10.0`?
